helper/testhelpers/mongodb/mongodbhelper.go

```diff
@@ -44,20 +40,14 @@ func PrepareTestContainerWithDatabase(t *testing.T, version, dbName string) (fun
 		if dbName != "" {
 			connURL = fmt.Sprintf("%s/%s", connURL, dbName)
 		}
-		dialInfo, err := ParseMongoURL(connURL)
-		if err != nil {
-			return nil, err
-		}
 
-		session, err := mgo.DialWithInfo(dialInfo)
+		ctx, _ = context.WithTimeout(context.Background(), 10*time.Second)
```

Comment:
I'm wondering if we should increase this timeout a bit? I see it was 1min before. A quick survey of other timeouts show most around ~1min.

Reply:
I increased it back to 1min